pkg/controllerutils/chaosdaemon.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controllerutils
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/chaos-mesh/chaos-mesh/controllers/config"
```

Comment:
Good! The dependency is much cleaner now!

But I still prefer to move this file out of `/pkg`, place it at `/controller/something` 🙈

Reply:
Nop. `/controller` contains only a lot of controllers (or "sub"controllers)

Reply:
Oops. As it already has some other things, I will move this pkg to `/controller/`

Reply:
Moved